extensions/nns/e2e/tests/nns.bash

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bats
 
-GIT_ROOT_DIR=$(git rev-parse --show-toplevel)
+export GIT_ROOT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
This means we don't run this script through shellcheck
```
In extensions/nns/e2e/tests/nns.bash line 3:
export GIT_ROOT_DIR=$(git rev-parse --show-toplevel)
       ^----------^ SC2155 (warning): Declare and assign separately to avoid masking return values.
```

Reply:
I fixed this one. I'll address the rest of lint errors in a separate PR #41